storage/series/series.go

```diff
@@ -401,26 +401,33 @@ func (s *dbSeries) bufferDrained(newBlock block.DatabaseBlock) {
 	// lock already. Executing the drain method occurs during a write if the
 	// buffer needs to drain or if tick is called and series explicitly asks
 	// the buffer to drain ready buckets.
-	s.mergeBlockWithLock(s.blocks, newBlock)
+	iOpts := s.opts.InstrumentOptions()
+	err := s.mergeBlockWithLock(newBlock)
+	if err != nil {
+		iOpts.Logger().WithFields(
+			xlog.NewField("id", s.id.String()),
+			xlog.NewField("blockStart", newBlock.StartTime()),
+			xlog.NewField("err", err.Error()),
+		).Errorf("error trying to drain series buffer")
+		// Allocating metric here is ok because this code-path should never
+		// happen anyways.
+		iOpts.MetricsScope().SubScope("series-buffer-drain")
```

Comment:
this is only creating a subscope, are you going to emit a metric for it too?

Reply:
...yes thank you. Broke this when I was refactoring